project.clj

```diff
@@ -93,7 +95,12 @@
                                        "examples/geometry/src"]
                         :compiler {:parallel-build true
                                    :main "reagentdemo.core"
-                                   :output-to "outsite/public/js/main.js"}}}}
+                                   :output-to "outsite/public/js/main.js"
+                                   :language-in :ecmascript6
+                                   :language-out :ecmascript3
+                                   :npm-deps {:react "15.6.1"
+                                              :react-dom "15.6.1"
+                                              :create-react-class "15.5.3"}}}}}
```

Comment:
Why `:create-react-class "15.5.3"` when above you have `:dependencies [cljsjs/create-react-class "15.6.0-1"]`?